src/Button.js

```diff
@@ -64,7 +68,11 @@ class Button extends React.Component {
       <Component
         {...this.props}
         type={this.props.type || 'button'}
-        className={classNames(this.props.className, classes)}>
+        className={classNames(this.props.className, classes)}
+        tabIndex="0"
+        role="button"
+        ariaLabel={this.props.ariaLabel}
```

Comment:
The default component here is a real `button` so the additions aren't actually needed in that case. A user _could_ specify a div, or some other component and then these would be really helpful, however we tend to leave that up to users to make the right choice, since adding the redundant roles and attributes (especially tabIndex) can have unintended side effects even for screen readers (at least in our experience). I'd say we could conditionally render them, but its not actually possible to know for sure what component the user will eventually render.